### PR TITLE
Added create/drop trigger dialect conditions

### PR DESCRIPTION
**Pull request by grosch**
Updates the package to specify syntax that PostgreSQL supports for triggers.

### CODE REVIEW

File: Sources/PostgresKit/PostgresDialect.swift [hunk 1]
```diff
@@ -37,4 +33,9 @@ public struct PostgresDialect: SQLDialect {
     public var enumSyntax: SQLEnumSyntax {
         .typeName
     }
+
+    public var triggerSyntax: SQLTriggerSyntax {
+        return .init(create: [.supportsForEach, .postgreSQLChecks, .supportsCondition, .conditionRequiresParentheses, .supportsConstraints],
```
Comment (tanner0101) [inline Sources/PostgresKit/PostgresDialect.swift]: Vapor style aligns indentation on new lines, e.g.:
```swift
Foo(
    bar: ...,
    baz: ...
)
```

File: docker-compose.yml [hunk 2]
```diff
@@ -1,35 +1,24 @@
-version: '2'
+version: '3.7'
 
-services:
-  psql-11:
-    image: postgres:11
-    environment:
+x-common: &common
```
Comment (tanner0101) [inline docker-compose.yml]: Nice 👍

File: Sources/PostgresKit/PostgresDialect.swift (hunk 1 above)
Comment (grosch) [reply Sources/PostgresKit/PostgresDialect.swift]: I'm not sure what you're asking me to do here.

Comment (grundoon) [reply Sources/PostgresKit/PostgresDialect.swift]: @grosch
```swift
    return init(
        create: [ ... ],
        drop: [ ... ]
    )
````

Comment (grosch) [reply Sources/PostgresKit/PostgresDialect.swift]: Thanks, done.

**Comment by tanner0101** [comment]
These changes are now available in [2.0.0-beta.3.6](https://github.com/vapor/postgres-kit/releases/tag/2.0.0-beta.3.6)

**Comment by tanner0101** [comment]
Thanks @grosch

**Pull request merged**